plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt

```diff
@@ -681,7 +683,21 @@ open class DefaultPageCreator(
                                         }
                                         after(extra = PropertyContainer.empty()) {
                                             contentForBrief(it)
-                                            contentForSinceKotlin(it)
+
+                                            val customTags = it.customTags
```

Comment:
For readability here it is possible to extract method.

Reply:
Sure, done